lib/sgds/sass/_variables.scss

```diff
@@ -1243,7 +1243,8 @@ $navbar-dark-brand-hover-color: $navbar-dark-active-color !default;
 // scss-docs-end navbar-theme-variables
 
 $masthead-height: 28px;
-
+$notify-banner-height: 46.8px;
+$portal-navbar-height: calc(#{$navbar-min-height} + #{$masthead-height} + #{$notify-banner-height});
```

Comment:
The purpose of adding this is because on the portal, the sidenav ([portal/assets/custom-css/components/_sidenav.scss](portal/assets/custom-css/components/_sidenav.scss)) and toc ([portal/assets/custom-css/base/_extends.scss](portal/assets/custom-css/components/_sidenav.scss)) is using the same height value for sticky position.

So I think we should have a single source of variable that controls the navbar height in case in the future we remove the notify banner we don't miss out resetting the height for either one of it.

But now I thought that maybe it's better to put this under portal's assets folder, @clukhei do you have any better suggestion for this?

Reply:
yea portal stuff has to be in portal/assets/custom-css/ folder. 

how about doing this inside the custom-css/components/_sidenav.scss

```
.sgds.sidenav.sticky{
    --sgds-sidenav-sticky-top: masthead height + navbar height + banner height
}
```

Reply:
i think this is good